### ISSUE TITLE
stricter distinctions needed between verification and validation

### ISSUE THREAD

**Issue by RieksJ** [decentralized-identity/presentation-exchange#168]
In the slipstream of #90, I would like us to consistently distinguish between verification and validation, where

The [Verifiable Credentials Data Model](https://www.w3.org/TR/vc-data-model/) has the following definitions:
- [verification](https://www.w3.org/TR/vc-data-model/#dfn-verify): “the evaluation of whether a verifiable credential or verifiable presentation is an authentic and timely statement of the issuer or presenter, respectively. This includes checking that: the credential (or presentation) conforms to the specification; the proof method is satisfied; and, if present, the status check succeeds”. 
- [validation](https://www.w3.org/TR/vc-data-model/#dfn-credential-validation): "The assurance that a verifiable credential or a verifiable presentation meets the needs of a verifier and other dependent stakeholders".

Obviously, verification is not validation. It seems to me that verification is a much more automatable activity than validation. It is about checking proofs, such as digital signatures. This suggests that verification can be made part of the SSI/IT infrastructure. 

Validation on the other hand cannot. The needs of a "verifier" (which we might want to rename into "validator") and other dependent stakeholders are highly subjective, and should come from the 'business-layer', e.g. as a (perhaps even machine-readable) policy. 

I think there is merit in discussing what it is that we actually need to distinguish between when we use terms such as 'validate/verify'. There may be merit in defining another SSI role, called 'validator', such that the 'verifier' role can still be performed by IT-only actors and the 'validator' role being more business oriented. In any case, properly distinguishing between these terms and using them as such in the document would certainly improve it.

**Comment by csuwildcat** [comment]
We talked about this on the call and feel that we can provide details about the differences in function between verification and validation under the actor description of the Verifier, noting that in some cases the Verifier may not directly perform the validation function. Would you be open to submitting a PR to this effect?

**Comment by henkvancann** [comment]
I've listened to the discussion during the call and made a draft graphical representation of the distinction between the two terms. Maybe it helps to clarify the nuances of the roles and functions of a verifier vs validator and where they apply.

Having drawn this infographic I discovered that a table might maybe even better :-|
<img width="681" alt="Screenshot 2021-01-07 at 19 56 19" src="https://user-images.githubusercontent.com/479356/103932667-75ade280-5122-11eb-860c-6318909f34d7.png">

**Comment by RieksJ** [comment]
I wasn't part of the discussion, but here's my/another 2c. The terms 'validation' and 'verification' need context to become meaningful. Verifying a (verifiable) credential is not the same as verifying (the truth of) a statement, or verifying an (electronic) identity. Validating a policy for the purpose of using it to make some kind of a decision is not the same as validating ideas you have for developing a product, or validating yourself. If you google with "what stuff can you validate?", you'll see there are many more ways in which these terms get different meanings.

Thus, I conclude that whenever a sentence contains words derived from verify/verification or validate/validation, I can only hallucinate about what the author is actually trying to say. And as these terms seem to have different meanings in different contexts, I think it is worth an effort ensure that whenever such terms are used -particularly in standards or specification documents- they refer to the (one of many possible) definition that makes sense in the context in which it is used. This may be a reference to an existing definition, or a definition that is provided in such documents themselves.

While every author could (and in my opinion should) do this, the hard thing is that they themselves would need to have a clear picture of what it is they try to convey, and I'm not sure that this is always the case...

**Comment by henkvancann** [comment]
Yes, the wording certainly needs context. E.g. in chapter 7 of the SSI book (ch7 by David Chadwick and Daniel Burnett), 'validator'/'validation' is absent. The topic of that chapter: VCs. They've chosen to use only 'verification' / 'verifier' so far (MEAP version).

As a non-native speaker, I don't want to mingle in discussions about wording in certain contexts in an expert field. I just took your initial text as an example to illustrate the possible use of graphical explanation in the context of VCs as I was listening in the discussion last night.

**Comment by bumblefudge** [comment]
I checked and I only saw one definitive malaprop here, addressed in one [this](https://github.com/decentralized-identity/presentation-exchange/commit/ed4599c4218563d5be3a4d44d44dd4d681e7b9aa) commit.  I think the verification/validation distinction is mostly OK!

In the spirit of editorial nitpicking, however, I'm going to close, then reopen and retag so that future editors can consider rewording the section [Validation of claims](https://identity.foundation/presentation-exchange/#validation-of-claims) to expand upon the distinction between "validation" (internal to Pres Exch messages, which is something like high-level or translational validation) and "subsequent validation". I think it might be clearer to refer to all translation handles by this protocol as some third term, or some special kind of validation? Too philosophical a question for now, but maybe for the future if the "internal validation" sections of the spec gets more complication or concrete.
